tests/README.md

```diff
@@ -35,7 +35,8 @@ Test all functions matching a name. For instance, this would pick up
 pytest -k add
 ```
 
-As a reminder, pytest can be helpful for debugging.
+As a reminder, pytest can be helpful for debugging. This command-lin option
+shows debug output from logging statements.
 
```

Comment:
We can use **pytest-parallel** package to run the test points in parallel; on a sample 8vcpu machine I see a savings of about 1/2 second when run with 10 workers
```
uthu@ubuntu-c-4-intel-sfo3-01:~/devel/openfhe-python$ pytest --workers 10
/home/muthu/.local/lib/python3.11/site-packages/pytest_parallel/__init__.py:221: PytestDeprecationWarning: The hookimpl ParallelRunner.pytest_sessionstart uses old-style configuration options (marks or attributes).
Please use the pytest.hookimpl(tryfirst=True) decorator instead
 to configure the hooks.
 See https://docs.pytest.org/en/latest/deprecations.html#configuring-hook-specs-impls-using-markers
  @pytest.mark.tryfirst
======================================= test session starts ========================================
platform linux -- Python 3.11.6, pytest-7.4.0, pluggy-1.2.0
rootdir: /home/muthu/devel/openfhe-python
configfile: pytest.ini
testpaths: tests
plugins: parallel-0.1.1, hypothesis-6.82.4
collected 13 items                                                                                 
pytest-parallel: 10 workers (processes), 1 test per worker (thread)
..............ss..ss......
================================== 11 passed, 2 skipped in 1.07s ===================================
~/devel/openfhe-python$ pytest 
======================================= test session starts ========================================
platform linux -- Python 3.11.6, pytest-7.4.0, pluggy-1.2.0
rootdir: /home/muthu/devel/openfhe-python
configfile: pytest.ini
testpaths: tests
plugins: parallel-0.1.1, hypothesis-6.82.4
collected 13 items                                                                                 

tests/test_bgv.py .........                                                                  [ 69%]
tests/test_ckks.py .                                                                         [ 76%]
tests/test_cryptocontext.py ss                                                               [ 92%]
tests/test_serial_cc.py .                                                                    [100%]

================================== 11 passed, 2 skipped in 1.64s ===================================
```

Reply:
That's a good idea. Openfhe, itself, isn't re-entrant. It stores data in static arrays and assumes there is only one thread that calls it. Internal, it uses OpenMP, but the programming model is that a single thread calls. I'm not sure parallel tests will always work. Can we save this for another PR when there are tests that do a bit more?